modules/system/controllers/Updates.php

```diff
@@ -176,22 +176,29 @@ public function details($urlCode = null, $tab = null)
         }
     }
 
-    protected function getPluginVersionFile($path, $filename)
+    protected function getPluginVersionFile($path, $filename): array
```

Comment:
Leave the return type hinting alone for now, we'll implement that across the entire code base all at once when we decide to do that.

Reply:
Ok